pkgs/by-name/mu/muse-sounds-manager/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  dpkg,
+  fontconfig,
+  zlib,
+  icu,
+  libX11,
+  libXext,
+  libXi,
+  libXrandr,
+  libICE,
+  libSM,
+  openssl,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "muse-sounds-manager";
+  version = "1.1.0.587";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20240826143936/https://muse-cdn.com/Muse_Sounds_Manager_Beta.deb";
```

Comment:
Hmm maybe you could explain why do you use a webarchive? Perhaps it'd be nice to also explain in a comment somewhere how to look up for new versions - in which website.

Reply:
Thanks for the suggestion! Done. http://web.archive.org/cdx/search/cdx?url=https://muse-cdn.com/Muse_Sounds_Manager_Beta.deb shows that the .deb actually has never changed since it was first archived in 2023